src/components/caption/index.jsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes from "prop-types";
+import formatCredits from "../../utils/format-credits";
+
+const COMPONENT_CLASS_NAME = "c-caption";
+
+const Caption = ({ caption, className, credit, title }) => {
+	const formattedCredit = formatCredits(credit);
```

Comment:
It was discussed that this utility would be for consumers to use and not directly built into the component

Reply:
updated!